infra/Pos/Subscription/Dns.hs

```diff
@@ -44,10 +45,15 @@ type KnownRelays = Map NodeId KnownRelay
 activeRelays :: KnownRelays -> [NodeId]
 activeRelays = map fst . filter (relayActive . snd) . M.toList
 
+-- TODO: Use valency and fallbacks
```

Comment:
Well I suppose there *is* a changing set of "best peers", but it changes sychronously with the subscriptions: we'll look for the next-best peer only if the current best peer is judged to be faulty because our subscription to it went away.